src/main/java/org/fcrepo/auth/webac/LinkHeaderProvider.java

```diff
@@ -71,7 +71,7 @@
                 new DefaultIdentifierTranslator(internalSession);
         final ListMultimap<String, String> headers = ArrayListMultimap.create();
 
-        LOGGER.debug("Adding WebAC Link Header for Resource: {}", resource);
+        LOGGER.debug("Adding WebAC Link Header for Resource: {}", resource.getPath());
```

Comment:
We aren't explicitly inheriting `getPath` from `javax.jcr.Item`: https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java#L46-L50, but the point is well taken: it's clearly JCR terminology: https://docs.adobe.com/docs/en/spec/javax.jcr/javadocs/jcr-2.0/javax/jcr/Item.html#getPath(). Maybe `getIdentifier` would be more appropriate?